### PR TITLE
Ebs encryption at rest

### PR DESCRIPTION
**Pull request by aliaksei-ivanou**
*Issue #, if available:*

*Description of changes:*
Creating a workload demonstrating EBS encryption-at-rest capabilities.

team-platform is responsible for creating an encrypted storage class `gp2-encrypted`. The provided customer managed KMS key is used for encryption, otherwise - a default AWS managed KMS key.

team-data deploys an example workload that uses the encrypted storage class via a PersistentVolumeClaim (PVC) `gp2-encrypted-pvc`. The same encryption KMS key is used.

Blueprint:
```
bootstrapValues: {
            spec: {
              kmsKey:
                "arn:aws:kms:us-east-1:111122223333:key/a8b9fa0b-955f-4f85-85c1-8f911003390e",
            },
          },
```

Results:
```
❯ kubectl get pvc -n data
NAME                STATUS   VOLUME                                     CAPACITY   ACCESS MODES   STORAGECLASS    AGE
gp2-encrypted-pvc   Bound    pvc-78bd070e-8eba-4b01-a378-462bb806beb3   10Gi       RWO            gp2-encrypted   14m

❯ aws ec2 describe-volumes --region us-east-1 --filters "Name=tag:kubernetes.io/created-for/pv/name,Values=pvc-78bd070e-8eba-4b01-a378-462bb806beb3" --query 'Volumes[*].{VolumeId:VolumeId, Encrypted:Encrypted, KmsKeyId:KmsKeyId}'

[
    {
        "VolumeId": "vol-09332f96a58e67385",
        "Encrypted": true,
        "KmsKeyId": "arn:aws:kms:us-east-1:111122223333:key/a8b9fa0b-955f-4f85-85c1-8f911003390e"
    }
]
```

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: teams/team-data/dev/templates/ebs-ecncrypted-storage-pvc.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
```
Comment (shapirov103) [inline teams/team-data/dev/templates/ebs-ecncrypted-storage-pvc.yaml:1]: Where is that PVC used? We discussed having a workload that shows mounting as well.

**Review by shapirov103 — COMMENTED** [review]
@aliaksei-ivanou please see my comment

File: teams/team-data/dev/templates/ebs-ecncrypted-storage-pvc.yaml (hunk 1 above)
Comment (aliaksei-ivanou) [reply teams/team-data/dev/templates/ebs-ecncrypted-storage-pvc.yaml:1]: Addressed

**Comment by shapirov103** [comment]
Validated functionally by @aliaksei-ivanou 

```
❯ kubectl get pods -o wide -n data

NAME                 READY   STATUS    RESTARTS   AGE   IP           NODE                           NOMINATED NODE   READINESS GATES
ebs-encryption-app   1/1     Running   0          54s   10.0.127.0   ip-10-0-125-125.ec2.internal   <none>           <none>
❯ kubectl exec ebs-encryption-app -n data -- bash -c "cat example/out.txt"
Wed Mar 22 18:07:47 UTC 2023
Wed Mar 22 18:07:52 UTC 2023
Wed Mar 22 18:07:57 UTC 2023
Wed Mar 22 18:08:02 UTC 2023
Wed Mar 22 18:08:07 UTC 2023
Wed Mar 22 18:08:12 UTC 2023
Wed Mar 22 18:08:17 UTC 2023
Wed Mar 22 18:08:22 UTC 2023
```

**Review by shapirov103 — APPROVED** [review]
lgtm

**Pull request merged**